www/app/hardware/Hardware.js

```diff
@@ -3916,7 +3917,9 @@ define(['app'], function (app) {
 							else if (HwTypeStr.indexOf("HTTP/HTTPS") >= 0) {
 								HwTypeStr += ' <span class="label label-info lcursor" onclick="CreateDummySensors(' + item.idx + ',\'' + item.Name + '\');">' + $.t("Create Virtual Sensors") + '</span>';
 							}
-							else if (HwTypeStr.indexOf("RFLink") >= 0) {
+							else if( (HwTypeStr.indexOf("RFLink") >= 0 &&
+									HwTypeStr.indexOf("MQTT") == -1 ) ||
```

Comment:
Do you need both MQTT checks here ?

HwTypeStr.indexOf("MQTT") == -1 ) ||
									HwTypeStr.indexOf("RFLink Gateway MQTT")

Should this not be HwTypeStr.indexOf("MQTT") != -1 )